pkg/cmd/server/kubernetes/master.go

```diff
@@ -63,8 +63,9 @@ func (c *MasterConfig) InstallAPI(container *restful.Container) []string {
 
 // RunNamespaceController starts the Kubernetes Namespace Manager
 func (c *MasterConfig) RunNamespaceController() {
-	// TODO: Add OR c.ControllerManager.EnableDeploymentController once we have upstream deployments
-	experimentalMode := c.ControllerManager.EnableExperimental
+	// we now have several of the kube "experimental" pieces enabled in Origin, so this needs to be
+	// enabled whenever we have the "experimental" APIs enabled.
+	experimentalMode := c.Master.EnableExp
```

Comment:
> this should logically key off of EnableExp, since we need to clean up those resources regardless of whether experimental controllers are running

I agree.  I also think its bad to have those two bool disagree.